### ISSUE TITLE
Is it possible to SET room temp?

### ISSUE THREAD

**Issue by msm8811** [sushilks/kumojs#16]
I am wondering if it's possible to use the kumojs interface to set the room temp the indoor unit is running against, so I can avoid buying a kumo remote temp unit?

**Comment by xkahn** [comment]
Weird bug report? Yes. You can set head temperatures. The README file shows the CLI commands to set heat and cool temps?

**Comment by msm8811** [comment]
No, this is not about setting the target temperature (what the unit is cooling/heating to), it's about setting the ambient temp in the room that the unit is tracking.  Right now, if I have a ducted unit, I have to use a Kumo wireless sensor for the unit to read the room ambient temp.  I'd love to be able to use a Home Assistant sensor or group of sensors to determine the ambient temp and send that to the unit.
